src/inference/include/openvino/runtime/common.hpp

```diff
@@ -50,3 +50,9 @@ namespace ov {
 using SupportedOpsMap = std::map<std::string, std::string>;
 
 }  // namespace ov
+
+#if defined(_WIN32) && !defined(__GNUC__)
```

Comment:
There is `ITT_FUNCTION_NAME` macro which looks more advance than this version. Can be used instead?